src/ripple/nodestore/impl/Database.cpp

```diff
@@ -25,12 +25,27 @@
 namespace ripple {
 namespace NodeStore {
 
-Database::Database(std::string name, Stoppable& parent,
-    Scheduler& scheduler, int readThreads, beast::Journal journal)
+std::uint32_t Database::genesisSeq_ {32570};
+
+Database::Database(
+    std::string name,
+    Stoppable& parent,
+    Scheduler& scheduler,
+    int readThreads,
+    Section const& config,
+    beast::Journal journal)
     : Stoppable(name, parent)
     , j_(journal)
     , scheduler_(scheduler)
 {
+    std::uint32_t gs;
+    if (get_if_exists<std::uint32_t>(config, "genesis_seq", gs))
+    {
+        if (genesisSeq_ != 32570)
+            Throw<std::runtime_error>("genesis_seq set more than once");
```

Comment:
How would a user set the genesis sequence more than once? Just trying to understand exactly what this checks against.

Reply:
With the Node store and the shard store.